README.md

```diff
@@ -67,16 +67,17 @@ The `polyfills.js` is packaged with this component, but **IT IS NOT NEEDED** to
 
 The `<auro-popover>` element should be used in situations where users may:
 
-* ...
-* ...
-* ...
+* Hover/blur over an element to display a tooltip-style graphic
 
 ## API Code Examples
 
 Default auro-popover
 
 ```html
-<auro-popover>Hello World</auro-popover>
+<button id="button1">Click me!</button>
```

Comment:
Are we saying anywhere in the use docs that you need a corresponding ID for a `for` reference?

Reply:
Updated in latest